package.json

```diff
@@ -33,7 +33,21 @@
     "gulp-live-server": "0.0.29",
     "gulp-mocha": "^2.2.0",
     "gulp-sww": "0.0.7",
+    "selenium-webdriver": "^2.52.0",
+    "karma": "^0.13.22",
+    "karma-chrome-launcher": "^0.2.3",
+    "karma-firefox-launcher": "^0.1.7",
+    "karma-mocha": "^0.2.2",
+    "karma-mocha-reporter": "^2.0.0",
+    "karma-requirejs": "^0.2.6",
+    "karma-chai": "^0.1.0",
+    "karma-chai-as-promised": "^0.1.2",
```

Comment:
I don't use promises in my simple tests, but we'll definitely deal with this a lot very soon, so including entire toolset